catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -34,6 +34,7 @@
 import javax.xml.transform.dom.DOMSource;
 import javax.xml.transform.stream.StreamSource;
 
+import org.apache.commons.lang.StringUtils;
```

Comment:
Doesn't look like the root pom had a dependency for  `commons-lang`. Also i forgot if we have the wrapped or servicemix commons-lang, but you'll need to embed it if we don't.